lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -274,6 +282,10 @@ div.course-wrapper {
       width: flex-grid(11.5) + flex-gutter();
     }
   }
+
+  @media print {
+    border: 0;
```

Comment:
Removes a border that normally prints around the course content, but doesn't look good when printed (especially since the right hand side of the border is usually off the page).